python/lib/sift_py/rule/service.py

```diff
@@ -227,3 +337,17 @@ class SubExpression:
     def __init__(self, fully_qualified_rule_name: str, expressions: Dict[str, Any]):
         self.fully_qualified_rule_name = fully_qualified_rule_name
         self.expressions = expressions
+
+
+class RuleChannelReference:
```

Comment:
Nit: This can just be a `dataclass` or `namedtuple`.

Reply:
Went with dataclass!